saleor/graphql/checkout/tests/mutations/test_checkout_lines_update.py

```diff
@@ -1622,3 +1622,181 @@ def add_variants_to_checkout_wrapper(*args, **kwargs):
     assert len(data["errors"]) == 1
     assert data["errors"][0]["field"] == "lineId"
     assert data["errors"][0]["code"] == CheckoutErrorCode.GRAPHQL_ERROR.name
+
+
+def test_checkout_lines_update_with_metadata(
+    user_api_client,
+    checkout_with_item,
+):
+    checkout = checkout_with_item
+    lines, _ = fetch_checkout_lines(checkout)
```

Comment:
I don't think that you need usage of `fetch_checkout_lines` in your test. It is just a wrapper on the `CheckoutLine`. 
You just need to confirm if the metadata was set. So you can just make `checkout.lines.first()` (as you're already doing). You can fully drop the `fetch_checkout_lines` from your tests - actually, you don't do anything with them.